test/pageObject/basePage/basePage.js

```diff
@@ -4,9 +4,6 @@ class BasePage {
   constructor() {
     this.Header = new Header();
   }
-  wait(waitInMilliseconds) {
-    return browser.sleep(waitInMilliseconds);
-  }
   open(url) {
     return browser.get(url);
```

Comment:
1) There is no need to `return` this statement.
2) Remember our conversation: there should be a separate property for each page: `this.url` (which will be unique for each page) and you should pass it INSIDE your `open()` method:
```
open() {
   browser.get(this.url);
}
```